casper/src/test/scala/coop/rchain/casper/MergeabilityRules.scala

```diff
@@ -198,7 +198,9 @@ trait MergeabilityRules {
       (expectOne(left) { findMatch(_, base) }, expectOne(right) { findMatch(_, base) }) match {
         case (Some(m1), Some(m2)) =>
           assert(
-            m1 == m2 && m1.maybeCardinality == Some(Linear) || m2.maybeCardinality == Some(Linear)
+            m1 == m2 && m1.maybeCardinality != Some(NonLinear) || m2.maybeCardinality != Some(
```

Comment:
wouldn't it make sense to introduce a few helpers?

eg. `def isNonLinear(m) = m1.maybeCardinality != Some(NonLinear)`

Reply:
`def isNonLinear(m) = m1.maybeCardinality.contains(NonLinear)`